Delegate_CategorySampleApp/ViewController.m

```diff
@@ -0,0 +1,78 @@
+//
+//  ViewController.m
+//  Delegate_CategorySampleApp
+//
+//  Created by webonise on 21/09/16.
+//  Copyright © 2016 webonise. All rights reserved.
+//
+
+#import "ViewController.h"
+
+@interface ViewController ()
+
+@end
+
+@implementation ViewController
+CustomAlertView *customAlertView;
+UIView *view1;
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view, typically from a nib.
+    customAlertView = [[CustomAlertView alloc]init];
+    customAlertView.delegate = self;
+    [customAlertView designAlertView :@"Do you want to delete?" :@"cancel" :@"OK"];
+    view1 = [customAlertView getAlertViewObject];
+    [self.view addSubview:view1];
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(notifyOtherController) name:NOTIFICATION_NAME object:nil];
+}
```

Comment:
in the assignment you asked us: On cancel, this alertView should dismiss.So on tapping  cancel only it dismisses.
I will add the button.